components/raftstore/src/coprocessor/config.rs

```diff
@@ -75,6 +81,8 @@ impl Default for Config {
             region_max_keys: SPLIT_KEYS / 2 * 3,
             consistency_check_method: ConsistencyCheckMethod::Mvcc,
             perf_level: PerfLevel::EnableCount,
+            enable_region_bucket: false,
+            region_bucket_size: DEFAULT_BUCKET_SIZE,
```

Comment:
should it consider the count of keys too?

Reply:
Today region has max keys count and split keys count, so I agree that bucket should have it as well.